exampleSite/content/posts/twemoji-support.md

```diff
@@ -0,0 +1,23 @@
++++
+date = "2020-05-25"
+title = "Twemoji Support 🤩"
+description = "The post demonstrates Twemooji support"
```

Comment:
You made a typo here, Twemoji, not Twemo**o**ji. 😉

Reply:
Twem😳ji